SparseGrids/tasgridExternalTests.cpp

```diff
@@ -2234,6 +2235,17 @@ bool ExternalTester::testAllAcceleration() const{
 void ExternalTester::debugTest(){
     cout << "Debug Test (callable from the CMake build folder)" << endl;
     cout << "Put testing code here and call with ./SparseGrids/gridtester debug" << endl;
+    int n = 200;
+    int nref = 501;
+    std::function<double(double)> sinc_plus1 =
```

Comment:
`std::function` is compatible with a lambda, but it is not really a lambda.

For these cases, you can just use auto:
```
auto sinc_pluis1 = [](double x)-> ....
```

Reply:
Done.